titus-common/src/main/java/com/netflix/titus/common/util/loadshedding/SpectatorAdmissionController.java

```diff
@@ -42,18 +44,18 @@ public AdmissionControllerResponse apply(AdmissionControllerRequest request) {
             AdmissionControllerResponse result = delegate.apply(request);
 
             registry.counter(METRIC_NAME,
-                    "callerId", request.getCallerId(),
-                    "endpointName", request.getEndpointName(),
+                    "callerId", getNonEmptyOrDefault(request.getCallerId(), "notSet"),
```

Comment:
minor: `s/notSet/unknown`

Reply:
I have purposefully chose different word, as we use `unknown` in many places in the code already. I want here something that is distinct, as we know that an empty value is a bug. Maybe better would be `requiredButNotSet`.